CHANGELOG.md

```diff
@@ -1,11 +1,9 @@
 # Changelog
 
 ## v2.1.1
-### Upgrade requirements
-### Migrations and checks
-#### Schema migrations
-#### Data migrations
+ 2025-07-28
```

Comment:
We only write specific date to the changelog in the release pull request.

If you want this PR to be release PR, please move the date change to separate commit and add to that commit also change in `version.txt`. Example: https://github.com/LayerManager/layman/pull/1145/commits/60f9ac32d66b2608fee8ab21fa3dee46ff3ce7b3